python/README.md

```diff
@@ -64,6 +57,14 @@ FusionEngine message specification.
           - This class is used by the `bin/p1_display` application
         - [file_reader.py](analysis/file_reader.py) - `FileReader` class, capable of loading and time-aligning
           FusionEngine data captured in a `*.p1log` file
+      - `applications` - Applications and processing tools
+        - [p1_display](bin/p1_display) - Generate HTML plots of vehicle trajectory, etc. (runs `analyzer.py`)
+        - [p1_extract](bin/p1_extract) - Extract FusionEngine message contents from a binary file containing mixed data
+          (e.g., interleaved RTCM and FusionEngine messages)
+        - [p1_lband_extract](bin/p1_lband_extract) - Extract L-band data bits contained in a log of FusionEngine
```

Comment:
isnt this in applications now?

Reply:
Good catch, fixed and reorganized this stuff a little better